src/dmd/mars.d

```diff
@@ -794,6 +694,107 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
     return status;
 }
 
+/**
+ * Parses the command line arguments and configuration files
+ */
+version(NoMain) {} else
+int parseCommandlineAndConfig(size_t argc, const(char)** argv, ref Param params, ref Strings files)
```

Comment:
why does it return int instead of bool?
can the ref parameters be out parameters instead?

Reply:
fixed